drf_writable_nested/mixins.py

```diff
@@ -115,6 +115,9 @@ def update_or_create_reverse_relations(self, instance, reverse_relations):
                 if related_data is None:
                     # Skip processing for empty data
                     continue
+                pk_name = field.Meta.model._meta.pk.attname
+                if pk_name not in related_data:
```

Comment:
The `related_data` can contain pk attribute instead of pk_name which usual equals to 'id'